Cargo.lock

```diff
@@ -25,15 +25,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bytes"
-version = "0.5.5"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-dependencies = [
- "loom 0.3.4 (registry+https://github.com/rust-lang/crates.io-index)",
-]
-
-[[package]]
-name = "cc"
-version = "1.0.54"
+version = "0.5.6"
```

Comment:
is it manual action or some `cargo`'s neat subcommand?

Reply:
I'm now trying to remember what I did. I suspect I reset the Cargo.lock back to master version and then re-ran `cargo build` or similar.

Reply:
;)